gatsby/src/pages/hosting.js

```diff
@@ -38,6 +38,19 @@ const Hosting = ({ data }) => {
                     in Linthal, Switzerland. They use an old building, second-hand servers, passive cooling and
                     are directly plugged into an on-site hydroelectric power plant!</p>
                 </div>
+
+                <div className="mxgrid__item50">
+                    <h2 id="ungleich">bamoid</h2>
+                    <p><img style={{"border-radius": "9px", "height": "130px"}} src="/images/bamoid_logo.jpg" alt="" /></p>
+                    <p><a href="https://www.bamoid.com">www.bamoid.com</a></p>
```

Comment:
Is this service still running? I just see a failed request when I visit https://www.bamoid.com which is what is linked?

http://bamoid.com/ goes to a Apache2 default page.